src/main/kotlin/racingcar/Race.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar
+
+class Race(numberOfCar: Int) {
+    val carList: List<Car> =
+        List(numberOfCar) { Car(0) }
```

Comment:
```suggestion
    val cars: List<Car> =
        List(numberOfCar) { Car(0) }
```

carList보다 cars로 표현하는게 더 어울리지 않을까요~?